src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
```

Comment:
It is not a good practice to create utility classes, instead of that you can create for example Database class and use `@Dao` annotation and injector added to the project or just create an instance of the class using `new` and the constructor.